stock_picking_mass_action/models/stock_picking.py

```diff
@@ -18,35 +18,32 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
+
 import logging
 
 from openerp.osv import orm
+from openerp.models import Model, api
```

Comment:
api is not inside openerp.models, but directly on openerp. Why not using usual form: `from openerp import models, api" and use `models.Model` for the class?